app/src/main/java/com/onelist/ui/MainActivity.kt

```diff
@@ -194,7 +195,11 @@ class MainActivity : ComponentActivity() {
     @Composable
     fun ItemList() { //Example List
         //TODO Will be replaced with a list of items
-        val listItems = listOf("Bread", "Cheddar Cheese", "Apples", "Toilet Paper", "Hand Soap")
+        val listItems: List<Item> = listOf(
```

Comment:
listItems should be a list of type <Item> not a list of strings.

Reply:
Yes.